### ISSUE TITLE
Accidentally dragged part 1 to FUSEE: now what?

### ISSUE THREAD

**Issue by putnam** [mattytrog/FUSEE_SUITE#11]
First, thanks for this project. Very handy for the Erista switches.

I have a Gemma M0.

I am trying to downgrade from v9 back to v6, because I always use Hekate, all my tools auto-update by replacing fusee.bin, and I don't want to keep renaming fusee.bin every time it is auto-updated.

I got into UF2 update mode, saw FUSEE as the drive, and I dragged in the old v6 Part 2. This sort of worked; I could get to the v6 boot menu by holding PWR and everything looked OK. I can also manually select a payload here and it boots OK.

But when it tries to cold boot into the payload from initial power on, I get weird behavior where the screen stays black.

I suspected that this was because I have the newer stage 1 bootloader on my Gemma, so I wanted to downgrade. But something seems to have changed in update mode where you no longer get the "SWITCHBOOT" drive first. So I didn't know how I was meant to replace the bootloader anymore.

So, then, stupidly, I dragged the old v6-era stage 1 UF2 file (FUSEE_SUITE_GEMMA.uf2) onto the FUSEE volume. After that the chip is basically bricked. It has one solid red light and one pulsing red light. Tapping the reset button doesn't do anything and no USB volumes mount anymore.

Any suggestions for what I should do next to repair this thing?

**Comment by putnam** [comment]
I found your post here:

**Comment by putnam** [comment]
Update: I found your post here on the GBATemp thread:
https://gbatemp.net/threads/internal-modchip-samd21-trinket-m0-gemma-m0-itsybitsy-m0-express-guide-files-support.508068/page-133#post-8564444

> Q - I cannot access SWITCHBOOT drive like I used to. Help!
> 
> A - Yep. Known problem. Just get to Switchboot Part 2 menu, double-press RESET so red LED is breathing, plug in USB cable, and boot something (cfw / ofw). SWITCHBOOT will pop up.

I used TegraRCMGUI to push Hekate over to the switch in this condition and got it to boot into the OS, and sure enough I got SWITCHBOOT back. That's a relief. I'm now reverting the bootloader back to the v6 edition.

**Comment by putnam** [comment]
Yeah, so, if I use the old Part 1, I can only ever get it to mount/remount SWITCHBOOT. FUSEE never mounts. If I upload the new Part 1, it mounts fine. I'm not sure what state I've gotten myself into here. Should it matter if I'm on the rewritten/newer part 1 and still running v6 for Part 2?

Right now I can get things going again by re-flashing v9. But I'd really like to get back to v6 if it's possible.

**Comment by mattytrog** [comment]
Hello. As long as you have a Fusee or switchboot drive show up in Windows,
you can upgrade. Just do part one then part 2. If you cannot get in to the
drive again, piwer down your switch, disconnect from usb, power on, connect
and double press the back cover (or whatever you have for Gemma reset), the
drive will pop up
